deploy/kubernetes/kiali-configmap.yaml

```diff
@@ -15,3 +15,8 @@ data:
         url: ${JAEGER_URL}
       grafana:
         url: ${GRAFANA_URL}
+    blacklist:
+      namespaces:
```

Comment:
see my original comment - let this file as-is and rely on a default. A user can add this section if they want to change the default blacklist. But if blacklist is not specified, we'll have a build-in default set of blacklist items.